profile/src/main/java/org/openedx/profile/presentation/profile/ProfileFragment.kt

```diff
@@ -344,6 +346,7 @@ private fun SupportInfoSection(
 ) {
     val uriHandler = LocalUriHandler.current
     val context = LocalContext.current
+    val config = getKoin().get<Config>()
```

Comment:
why config is injected here, it need to be injected inside viewModel, and all config fields should be provided throw uiState into UI